tests/test_searching.py

```diff
@@ -0,0 +1,44 @@
+"""Module to test the Sorting algorithms."""
+
+
+from unittest import TestCase
+
+from parameterized import parameterized
+
+from algorithms.searching.search import Search
+from algorithms.searching.lineal_search import LinealSearch
+from algorithms.searching.binary_search import BinarySearch
+
+
+class TestSearch(TestCase):
+    """Base class to test Sorting implementations."""
+
+    @parameterized.expand(
+        [
+            (
+                LinealSearch,
+                lambda x, y: x - y,
```

Comment:
I'm still having doubts about this type of comparison functions. Should we agree on trying to make them return a `boolean` value maybe?